litellm/__init__.py

```diff
@@ -557,7 +560,7 @@ def add_known_models():
     "anyscale",
     "mistral",
     "groq",
-    "nvidia_nim",
```

Comment:
people have been using "nvidia_nim". instead of removing it, can it be an alias for "nvidia"?